src/elements/skiplink-list/skiplink-list.ts

```diff
@@ -49,7 +50,7 @@ class SbbSkiplinkListElement extends SbbNamedSlotListMixin<
 
     if (changedProperties.has('listChildren')) {
       for (const child of this.listChildren) {
-        child.size = 'm';
+        child.size = isLean() ? 'xs' : 'm';
```

Comment:
Not sure whether the lean design should be applied here. I'ts only visible on focus and doesn't matter which size it has. Probably in business applications it won't be used at all. What do you think @mcilurzo ?